### PR TITLE
Update inline_image_push.md

### PR DESCRIPTION
**Pull request by davidbielik**
clarifying requirements and image of the push type dropdown

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 1]
```diff
@@ -13,11 +13,17 @@ This feature is in Early Access. Please ask your account manager if you'd like t
...
 ![Android Inline Image Push example]({% image_buster /assets/img/android/push/inline_image_push_android_1.png %}){: style="float:right;max-width:30%;margin-left:15px;border: 0;"}
 
-> Inline Image push notifications are custom push notifications by Braze that show images inline without the need to manually expand like standard push notifications.
+> Use the Inline Image push type to showcase your push notification images so users won't have to manually expand the message as required with a standard push type.
 
-# Usage Overview
+# Usage Requirements
 
-No extra integration is required for Android Inline Image Push after upgrading to Braze Android SDK 10.0.0. Inline Image push notifications require Android M+ devices. Devices lower than Android M will see standard big image push notifications instead of inline images. Additionally, push action buttons are not supported when using this notification type.
+This new notification type requires the Braze Android SDK v10.0.0 or greater, and Android M+ devices or newer.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:20]:
```suggestion
This new notification type requires the Braze Android SDK v10.0.0+, and Android M+ devices.
```

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 2]
```diff
@@ -13,12 +13,19 @@ This feature is in Early Access. Please ask your account manager if you'd like t
...
 ![Android Inline Image Push example]({% image_buster /assets/img/android/push/inline_image_push_android_1.png %}){: style="float:right;max-width:30%;margin-left:15px;border: 0;"}
 
-> Inline Image push notifications are custom push notifications by Braze that show images inline without the need to manually expand like standard push notifications.
+> Showcase your push notification's image using an Inline Image push so users won't have to manually expand a standard push to enlarge its image.
 
-# Usage Overview
+# Usage Requirements
+
+{% alert tip %}
+No additional integration or SDK changes are required to use this feature!. Devices or SDKs which don't meet the minimum version requirements will instead show a standard big image push notifications.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:21]:
```suggestion
No additional integration or SDK changes are required to use this feature!. Devices or SDKs which don't meet the minimum version requirements will instead show a standard big image push notification.
```

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 3]
```diff
@@ -13,12 +13,19 @@ This feature is in Early Access. Please ask your account manager if you'd like t
 
 ![Android Inline Image Push example]({% image_buster /assets/img/android/push/inline_image_push_android_1.png %}){: style="float:right;max-width:30%;margin-left:15px;border: 0;"}
 
-> Inline Image push notifications are custom push notifications by Braze that show images inline without the need to manually expand like standard push notifications.
+> Showcase your push notification's image using an Inline Image push so users won't have to manually expand a standard push to enlarge its image.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:16]:
```suggestion
> Showcase a larger image within your Android push notification using Inline Image push. With this design, users won't have to manually expand the push to enlarge the image.
```

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 4]
```diff
@@ -13,12 +13,19 @@ This feature is in Early Access. Please ask your account manager if you'd like t
...
 
-# Usage Overview
+# Usage Requirements
+
+{% alert tip %}
+No additional integration or SDK changes are required to use this feature!. Devices or SDKs which don't meet the minimum version requirements will instead show a standard big image push notifications.
+{% endalert %}
+
+- This notification type requires the Braze Android SDK v10.0.0+, and Android M+ devices. Unsupported devices or SDKs will fallback to the standard big image push notifications
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:24]:
```suggestion
- This notification type requires the Braze Android SDK v10.0.0+ and Android M+ devices. Unsupported devices or SDKs will fallback to the standard big image push notification.
```

**Review by radixdev — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 5]
```diff
@@ -13,12 +13,19 @@ This feature is in Early Access. Please ask your account manager if you'd like t
...
 ![Android Inline Image Push example]({% image_buster /assets/img/android/push/inline_image_push_android_1.png %}){: style="float:right;max-width:30%;margin-left:15px;border: 0;"}
 
-> Inline Image push notifications are custom push notifications by Braze that show images inline without the need to manually expand like standard push notifications.
+> Showcase a larger image within your Android push notification using Inline Image push. With this design, users won't have to manually expand the push to enlarge the image.
 
-# Usage Overview
+# Usage Requirements
+
+{% alert tip %}
+No additional integration or SDK changes are required to use this feature!. Devices or SDKs which don't meet the minimum version requirements will instead show a standard big image push notifications.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:21]:
```suggestion
No additional integration or SDK changes are required to use this feature!. Devices or SDKs which don't meet the minimum version requirements will instead show a standard big image push notification.
```

**Review by timothy-kim — APPROVED** [review]
lgtm 👌

**Pull request merged**